extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -149,6 +165,14 @@ class AmpIframe extends AMP.BaseElement {
     iframe.src = this.iframeSrc;
     this.element.appendChild(makeIOsScrollable(this.element, iframe));
 
+    (function(parentContext) {
```

Comment:
needed to call activateIframe with the context of the class - else i would not be able to call this.activateIframe , if i use fat arrow function then this.readyState = 'complete'; (this is the iframe)